src/Microsoft.Data.Analysis/DataFrame.IO.cs

```diff
@@ -269,54 +272,54 @@ private static DataFrameColumn CreateColumn(Type kind, string[] columnNames, int
                 }
                 columns.Insert(0, indexColumn);
             }
+
+            textReader.Dispose();
```

Comment:
Use `using` instead.

Reply:
We're instantiating `textReader` twice though. I wrapped the last `textReader` var into a `using` block now. It just looks a little odd is all.

Reply:
You should be able to dispose both since we use `leaveOpen`.